### PR TITLE
Add 2 new cases for hyper-v flags

### PR DESCRIPTION
**Pull request by peixiu**
1. cpuid check with hv-tlbflush-ext flag
2. cpuid check with hv-tlbflush-direct flag

ID: 1106, 1107

Signed-off: Peixiu Hou phou@redhat.com

### CODE REVIEW

**Comment by peixiu** [comment]
python3 ConfigTest.py --testcase=hv_flag_cpuid_check --guestname=RHEL.9.3.0 --platform=x86_64 --firmware=default_bios --clone=no --machines=q35

 (1/2) Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.3.0.x86_64.io-github-autotest-qemu.hv_flag_cpuid_check.hv_tlbflush_ext.q35: STARTED
 (1/2) Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.3.0.x86_64.io-github-autotest-qemu.hv_flag_cpuid_check.hv_tlbflush_ext.q35: PASS (133.46 s)
 (2/2) Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.3.0.x86_64.io-github-autotest-qemu.hv_flag_cpuid_check.hv_tlbflush_direct.q35: STARTED
 (2/2) Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.3.0.x86_64.io-github-autotest-qemu.hv_flag_cpuid_check.hv_tlbflush_direct.q35: PASS (134.72 s)

@menli820 @YongxueHong Could you help to review this patch?

Thanks~
Peixiu

**Comment by YongxueHong** [comment]
Hi @peixiu 
Looks like the CI checking has failed.

File: qemu/tests/cfg/hv_flag_cpuid_check.cfg [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        - hv_tlbflush_ext:
+            hv_flag = "hv_tlbflush_ext"
+            cpu_model_flags += ",hv-tlbflush-ext,hv-tlbflush,hv-vpindex"
+            check_cpuid_entry_cmd = "cpuid -r -1 -l 0x40000003"
+            check_register = "edx"
+            check_bit = 14
+            hv_flags_to_ignore = hv-tlbflush-ext
+        - hv_tlbflush_direct:
+            hv_flag = "hv_tlbflush_direct"
+            cpu_model_flags += ",hv-tlbflush-direct,hv-vapic"
```
Comment (menli820) [inline qemu/tests/cfg/hv_flag_cpuid_check.cfg:18]: @peixiu  we don't have to add hv-vapic here for it is a default flag already added in avocado.

internal_cfg/qemu-build/qemu-kvm.cfg:13:        cpu_model_flags += ",hv_stimer,hv_synic,hv_vpindex,hv_relaxed,hv_spinlocks=0x1fff,**hv_vapic**,hv_time,hv_frequencies,hv_runtime"

Comment (peixiu) [reply qemu/tests/cfg/hv_flag_cpuid_check.cfg:18]: Hi @menli820 

Thanks for you propoal, It seems AMD has some issues, none this flag on the AMD host, so I added it here, in order to make sure this case needed hv-flag all are attached. These 2 flags is bounded for this case.

Thanks~
Peixiu

**Comment by peixiu** [comment]
> Hi @peixiu Looks like the CI checking has failed.

Updated, thanks~

File: qemu/tests/cfg/hv_flag_cpuid_check.cfg (hunk 1 above)
Comment (menli820) [reply qemu/tests/cfg/hv_flag_cpuid_check.cfg:18]: As we talked privately， it is reasonable. so no changed need.

**Comment by menli820** [comment]
LGTM

**Comment by menli820** [comment]
LGTM

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Pull request merged**